client/src/app/payroll/payroll-list/payroll-list.component.html

```diff
@@ -4,18 +4,15 @@
       <th scope="col">Name</th>
       <th scope="col">Payroll Period</th>
       <th scope="col">Check Date</th>
-      <th scope="col">Amount</th>
+      <th scope="col">Gross Amount</th>
       <!-- <th scope="col">Risk</th> -->
     </tr>
   </thead>
   <tbody>
-    <tr
-      [ngClass]="{
-        'table-secondary': payroll.employeeStatus === 'Disabled'
-      }"
-      *ngFor="let payroll of getPayroll()"
-    >
-      <th scope="row"> {{ payroll.employeeName }} </th>
+    <tr *ngFor="let payroll of payrolls | async">
+      <th scope="row">
+        {{ payroll.employeeFirstName + ' ' + payroll.employeeLastName }}
```

Comment:
You can also `{{ payroll.employeeFirstName }} {{ payroll.employeeLastName }}`

Reply:
fixed